File: teledonJFX/src/main/java/com/example/teledonjfx/service/VolunteerService.java
package com.example.teledonjfx.service;

import com.example.teledonjfx.model.Volunteer;
import com.example.teledonjfx.repository.IVolunteerRepository;
import com.example.teledonjfx.repository.VolunteerRepository;

public class VolunteerService {
    private final IVolunteerRepository volunteerRepository;

    public VolunteerService(VolunteerRepository volunteerRepository) {
        this.volunteerRepository = volunteerRepository;
    }

    public Volunteer getVolunteer(String username,String password){
        return volunteerRepository.getByUsernameAndPassword(username,password);
    }
}

File: teledonDesktop/teledonDesktop/services/VolunteerService.cs
using System;
using teledonCS.model;
using teledonCS.repository;

namespace teledonDesktop.services
{
    public class VolunteerService
    {
        public IVolunteerRepository VolunteerRepository;

        public VolunteerService(IVolunteerRepository volunteerRepository)
        {
            VolunteerRepository = volunteerRepository;
        }

        public Volunteer getVolunteer(String username, String password)
        {
            return VolunteerRepository.getByUsernameAndPassword(username, password);
        }
    }
}